lib/main.js

```diff
@@ -50,11 +60,14 @@ class CheckEmptyPayload extends Transform {
     }
 }
 
+const ALLOWED_TYPES = ['js', 'css'];
+
 module.exports = class Router extends EventEmitter {
-    constructor(sink) {
+    constructor(sink, logger = { info() {}, error() {} }) {
```

Comment:
is the logger needed? we emit events now - why not stick to that pattern?

Reply:
Now that you mention it, that does seem a bit inconsistent. Maybe sticking with emitting for now until we are clear about our logging/debugging story is wise.

Reply:
fixing...